backend/uclapi/timetable/create_models.py

```diff
@@ -0,0 +1,99 @@
+file_string = open("Full_DDL.sql").read()
+# file_string = open("test.txt").read()
```

Comment:
Remove this.

Reply:
This is an ugly script. Dont review haha